.travis.yml

```diff
@@ -1,13 +1,30 @@
+dist: trusty
+sudo: false
+
 language: node_js
-node_js: stable
-branches:
-  only:
-    - master
-    - /^greenkeeper\/.*$/
+
+node_js:
+  - "7"
```

Comment:
You right for the node v7 version, I could replace it by the v6 one. 

This library mainly target node user, these test are to ensure our code does not break between node major release. But you right it's maybe too much since we do have that many code that use the node API directly. I will remove it and only test against the latest version